### PR TITLE
chore(ci): use GitHub Actions instead of TravisCI

### PR DESCRIPTION
**Pull request by oltodo**
Make CI easier to use and configure by using GitHub Actions instead of TravisCI.

### CODE REVIEW

**Review by valotvince — APPROVED** [review]
😻

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (valotvince) [inline .github/workflows/ci.yml:7]: Don't know the notation, but it should be on all PRs, nope ?
```suggestion
    branches: *
```

Comment (oltodo) [reply .github/workflows/ci.yml:7]: Good point. That comes from the [Node.js CI starter template](https://github.com/actions/starter-workflows/blob/main/ci/node.js.yml) provided by GitHub. But surprisingly, the CI works fine on that branch.

Comment (oltodo) [reply .github/workflows/ci.yml:7]: Ok I figured it out 🤯. GitHub Actions evaluate the **base** branch to determine if the workflow must be launched.

**Review by Slashgear — APPROVED** [review]

**Review by fdubost — APPROVED** [review]
👍

**Pull request merged**